ui/job-view/pushes/Push.jsx

```diff
@@ -305,14 +304,32 @@ class Push extends React.Component {
 
     try {
       const decisionTaskId = await getGeckoDecisionTaskId(push.id, repoName);
-      const jobList = await RunnableJobModel.getList(repoName, {
```

Comment:
I think it would be good to move the changes here into `RunnableJobModel`, so as to keep this file as small as possible

Reply:
Something like this?